martian-derive/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "martian-derive"
-version = "0.24.0"
+version = "0.25.0"
```

Comment:
I think it's a bad idea to break semver compatibility when there's no actual change.